draconic/helpers.py

```diff
@@ -40,6 +42,8 @@ def __init__(self, max_const_len=200000, max_loops=10000, max_statements=100000,
         self.max_statements = max_statements
         self.max_power_base = max_power_base
         self.max_power = max_power
+        self.min_int = -(2 ** max_int_size)
+        self.max_int = (2 ** max_int_size) - 1
```

Comment:
Should definitely cap this - `2**1024` is fast, `2**1209301293` has been chomping a CPU core for a while now 😅

Reply:
This config object is never exposed to a user - I guess it's a good anti foot-gun? It's up to the consumer of the interpreter to provide sane values to configure the interpreter, though